src/data/finset/basic.lean

```diff
@@ -611,12 +642,17 @@ lemma coe_union (s₁ s₂ : finset α) : ↑(s₁ ∪ s₂) = (s₁ ∪ s₂ :
 theorem union_subset {s₁ s₂ s₃ : finset α} (h₁ : s₁ ⊆ s₃) (h₂ : s₂ ⊆ s₃) : s₁ ∪ s₂ ⊆ s₃ :=
 val_le_iff.1 (ndunion_le.2 ⟨h₁, val_le_iff.2 h₂⟩)
 
+lemma union_subset_iff {s₁ s₂ s₃ : finset α} :
+  s₁ ∪ s₂ ⊆ s₃ ↔ s₁ ⊆ s₃ ∧ s₂ ⊆ s₃ :=
+⟨λ h, ⟨λ x hx, h (mem_union_left _ hx), λ x hx, h (mem_union_right _ hx)⟩, λ ⟨h₁, h₂⟩,
+  union_subset h₁ h₂⟩
```

Comment:
Note that when you import `data.finset`, you automatically get this lemma from `sup_le_iff`, so this isn't really a hole; it's more like a duplicate.